de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/table/api/impl/CopyColumnsProducer.java

```diff
@@ -220,4 +222,16 @@ private I_AD_Column copyColumn(final I_AD_Column sourceColumn)
 
 		return colTarget;
 	}
+	
+	private void disallowLogging(@NonNull final I_AD_Column sourceColumn, @NonNull final I_AD_Column colTarget)
```

Comment:
i think having only colTarget param is enough. U can get the ColumnName for it.

Reply:
i think it would be better to refactor this method, introduce it in some BL and use it everywhere.
It could be called something like "boolean getDefaultAllowLoggingByColumnName(final String columnName)"...

Reply:
hm.. yes!